### ISSUE TITLE
Incompatible with next.js 9.3

### ISSUE THREAD

**Issue by BjoernRave** [vinissimus/next-translate#66]
When trying to use this lib with next.js 9.3 I get the following error:

```
[1] internal/modules/cjs/loader.js:796
[1]     throw err;
[1]     ^
[1] 
[1] Error: Cannot find module '@babel/runtime-corejs2/helpers/interopRequireDefault'
[1] Require stack:
[1] - /Users/bjoern/projects/inventhora/frontend/node_modules/next-translate/_helpers/getPageNamespaces.js
[1] - /Users/bjoern/projects/inventhora/frontend/node_modules/next-translate/cli/builder.js
[1]     at Function.Module._resolveFilename (internal/modules/cjs/loader.js:793:17)
[1]     at Function.Module._load (internal/modules/cjs/loader.js:686:27)
[1]     at Module.require (internal/modules/cjs/loader.js:848:19)
[1]     at require (internal/modules/cjs/helpers.js:74:18)
[1]     at Object.<anonymous> (/Users/bjoern/projects/inventhora/frontend/node_modules/next-translate/_helpers/getPageNamespaces.js:1:41)
[1]     at Module._compile (internal/modules/cjs/loader.js:955:30)
[1]     at Object.Module._extensions..js (internal/modules/cjs/loader.js:991:10)
[1]     at Module.load (internal/modules/cjs/loader.js:811:32)
[1]     at Function.Module._load (internal/modules/cjs/loader.js:723:14)
[1]     at Module.require (internal/modules/cjs/loader.js:848:19) {
[1]   code: 'MODULE_NOT_FOUND',
[1]   requireStack: [
[1]     '/Users/bjoern/projects/inventhora/frontend/node_modules/next-translate/_helpers/getPageNamespaces.js',
[1]     '/Users/bjoern/projects/inventhora/frontend/node_modules/next-translate/cli/builder.js'
[1]   ]
[1] }
```

**Comment by BjoernRave** [comment]
never mind, just saw, that you released a new version for that :D That was fast :)

**Comment by aralroca** [comment]
@BjoernRave yes, the polyfill optimization feature (shipped now with 9.3) was breaking the build. However, I did a canary version of `next-translate` some days before in order to support this feature when it was experimental. Now I just deployed as final version instead of canary 👍
